.changeset/tasty-scissors-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@atproto/api': patch
+---
+
+Testing out changesets
```

Comment:
If merged, this should result in a `patch` for `api`, an update of the `api` package in each package that relies on it, and a `patch` of their versions as well.

In the `CHANGELOG.md`, this comment will appear along with the git hash so that changes can be easily linked to the actual edited files.

Reply:
Removed this, but you get the idea